Readme.md

```diff
@@ -0,0 +1,73 @@
+# SellEverywhere
+
+## Overview
+We want to create a hardware shop using popular technologies in a friendly team
+(MKdir). Examples are any store selling goods. Current idea is to implement an
+admin panel and user panel. The basic interface of the store was designed in [Figma](https://www.figma.com/file/Re4tXW7aNCWOzed7wI1GKe/dirStore).
+
+### Basic flow
+![Basic flow](./docs/basic_flow.png)
+
+### The application must have the following functionality from the administrator perspective:
+
+* authorization - jwt;
+* authentication;
+* creation of a new product (price, description, category, name, manufacturer, photo);
+* manage existing products/deliverers (read/edit/delete);
+* add new products/deliverers;
```

Comment:
one product can have variants with different price

Reply:
No, just product can be delivered by specified deliverers and they propose different price for delivering